src/app/src/map.actions.js

```diff
@@ -1,6 +1,18 @@
 import { createAction } from 'redux-act';
+import { FlyToInterpolator } from 'react-map-gl';
 
 export const toggleBackToMapButton = createAction(
     'Toggle back to map button visibility'
 );
-export const updateViewport = createAction('Update map viewport');
+export const updateViewport = createAction('Update map viewport', options =>
```

Comment:
Didn't realize that these could have functions as their second arguments!

Reply:
Yeah, the docs say it's for merging multiple arguments into one. Works well in this case.